doc/source/security/encrypt-traffic.rst

```diff
@@ -61,54 +63,62 @@ Replication traffic refers to the inter-node traffic which includes
 
 Traffic of each type is transferred via different channel, and so it is
 important to configure secure channels for all 3 variants to completely
-secure the replication traffic.
-
-|PXC| supports a single configuration option which helps to
-secure complete replication traffic, and is often referred as Automatic
-Configuration. User can also ignore this and configure security of
-each channel by specifying independent parameters.
+secure the replication traffic. 
 
-Section below will help, covering this aspect.
+|PXC| supports a single configuration option which helps to secure the complete
+replication traffic, and is often referred to as :ref:`ssl-auto-conf`. You can
+also configure the security of each channel by specifying independent
+parameters.
 
 .. _ssl-auto-conf:
 
 SSL Automatic Configuration
 ===========================
 
+The automatic configuration of the SSL encryption needs a key and certificate
+files. MySQL generates a default key and certificate files and places them in
+the data directory. It is important that your cluster use the same SSL
+certificates on all nodes.
```

Comment:
It will be great if we highlight this `It is important that your cluster use the same SSL
certificates on all nodes.`